lib/html-to-vdom.js

```diff
@@ -3,8 +3,23 @@ var parseHTML = require('./parse-html');
 
 module.exports = function initializeHtmlToVdom (VTree, VText) {
     var htmlparserToVdom = createConverter(VTree, VText);
-    return function convertHTML(html) {
+    return function convertHTML(html, options) {
+    	// Allow specifying the options as the first argument to allow binding of this function. 
+    	// This is useful to support specifying options to all calls with a single bound function.
+    	if (typeof html !== 'string' && options) {
+    		var args = Array.prototype.slice.call(arguments);
+    		options = args[0];
+    		html = args[1];
+    	}
+
         var tags = parseHTML(html);
-        return htmlparserToVdom.convertTag(tags[0]);
+
+        if (options && options.getVNodeKey){
+        	htmlparserToVdom.getVNodeKey = options.getVNodeKey;
```

Comment:
I would prefer to pass this explicitly as a second argument to `convertTag`.
`htmlparserToVdom` is shared between all `convertHTML` calls for the entirety of the converter's existence. I don't like attaching things to it, even if we remove it again later.

If in the future we have to go back to an asynchronous API for some reason, this code could break badly.